[Private preview] Add docs for Cloud Connect + AutoOps for self-managed clusters (#2287)
elastic/docs-content: deploy-manage/monitor/autoops/cc-manage-users.md

---
applies_to:
  deployment:

>     self:

Reviewer: ```suggestion
    self:
    eck:
    ece:
```

navigation_title: Manage users
---

# Manage users

Learn how to invite users to your connected clusters and assign roles to new and existing users.

## Invite users

:::{note}
:::{include} /deploy-manage/monitor/_snippets/single-cloud-org.md
:::
:::

To invite users to your organization and give them access to your self-managed cluster:

1. Log in to [{{ecloud}}](https://cloud.elastic.co/home).
2. In the **Connected clusters** section, select a cluster.
3. From the lower navigation menu, select **Organization**. 
4. On the **Members** page, click **Invite members**.
5. Enter the email address of the user you want to invite in the textbox. \

> To add multiple users, enter their email addresses separated by a space.

Reviewer 2: Are the `\`s intentional here and below? I typically use indentation to create a separate paragraph, and in that case I think it renders differently, with space between the lines. This is probably okay too but I just wanted to make sure it's not something pulled in by accident via copy/paste.

<img width="548" height="364" alt="screen" src="https://github.com/user-attachments/assets/e17e9137-5284-4125-b680-37b1ee28b8a6" />

Author: It was an error. Thanks for pointing it out! Fixed

> 6. In the **Assign roles** section, switch on **Connected cluster access**. 

Reviewer 2: ```suggestion
6. In the **Assign roles** section, enable **Connected cluster access**. 
```

7. Set roles for the user(s) on all or selected self-managed clusters so that they have the appropriate permissions when they accept the invitation and sign in to {{ecloud}}. Learn more about roles and their levels of access to AutoOps in [Assign roles](#assign-roles).
8. Click **Send invites**. \

> Invitations to join an organization are sent by email. Invited users have 72 hours to accept the invitation before it expires. If the invite has expired, an admin can resend the invitation.

Reviewer 2: ```suggestion
Invitations to join an organization are sent by email. Invited users have 72 hours to accept the invitation before it expires. If an invitation expires, an admin can resend it.
```
